docs/plugins/authors.asciidoc

```diff
@@ -119,3 +119,35 @@ You may also load your plugin within the test framework for integration tests.
 Read more in {ref}/integration-tests.html#changing-node-configuration[Changing Node Configuration].
 
 
+[float]
+=== Java Security permissions
```

Comment:
I wonder if we should add a discouraging paragraph here that makes folks to think twice about adding exceptions? I am not sure how  it should be phrased just putting the question out there...